app/views/catalog/_index_header_list_default.html.erb

```diff
@@ -1,10 +1,7 @@
-<%# OVERRIDE Hyrax v5.0.0rc2 to support shared search %>
-<% model = document.hydra_model %>
+<%# OVERRIDE Hyrax v5.0.1 to collection badge and markdown of links %>
```

Comment:
are we sure these two do not conflict? I feel like maybe we are breaking something in shared search here